app/views/admins/categories/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class='container is-fluid'>
+  <nav class="level is-mobile">
+    <div class="level-left">
+      <p class='is-size-3'><strong>Categories</strong></p>
+    </div>
+    <div class="level-right">
+      <%= link_to 'New Category', new_admins_category_path, class: 'button is-primary level-item' %>
+    </div>
+  </nav>
+  <div class='box'>
+    <div class="list has-hoverable-list-items has-visible-pointer-controls">
+      <% @categories.each do |category| %>
```

Comment:
maybe `includes(:rewards)` would be useful here? There is `category.rewards.count` which can cause n+1 query problems